ticketconfig.py

```diff
@@ -52,9 +52,13 @@ def _setup_providers(self):
             ))
         p.append( h.GitlabTitleProvider( 'gitlab.torproject.org',
             'https://gitlab.torproject.org/',
-            prefix='gitlabtpo:',
+            prefix='tor:',
             default_re=r'(?<!\w)(?:gitlabtpo:)(?P<path>\w+/[\w/]*\w)#(?P<number>[0-9]+)(?:(?=\W)|$)',
             ))
```

Comment:
Same as in the comment about the tractpo provider, prefix and regex probably should match

Reply:
ah yes. the point here was to reuse the `tor` prefix.